CMakeLists.txt

```diff
@@ -1,13 +1,29 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.5)
 project(python_qt_binding)
 
-find_package(catkin REQUIRED)
-catkin_package(CFG_EXTRAS python_qt_binding-extras.cmake)
+find_package(ament_cmake REQUIRED)
+find_package(ament_cmake_python REQUIRED)
 
-catkin_python_setup()
+ament_python_install_package(${PROJECT_NAME})
 
+set(python_qt_binding_CMAKE_DIRS cmake)
 install(FILES
   cmake/shiboken_helper.cmake
   cmake/sip_configure.py
   cmake/sip_helper.cmake
-  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}/cmake)
+  DESTINATION share/${PROJECT_NAME}/cmake/)
+
+  if(BUILD_TESTING)
+    find_package(ament_cmake_pytest REQUIRED)
+    find_package(ament_lint_auto REQUIRED)
+    ament_lint_auto_find_test_dependencies()
+
+    ament_add_pytest_test(pqt_test test
+      PYTHON_EXECUTABLE "${PYTHON_EXECUTABLE}"
+      APPEND_ENV AMENT_PREFIX_PATH=${ament_index_build_path}
```

Comment:
Where is the variable `ament_index_build_path` being defined?

Reply:
Leftover from a copy-paste job. Removed append to AMENT_PREFIX_PATH